alcs-api/src/application/application-meeting/application-meeting.service.ts

```diff
@@ -51,17 +52,63 @@ export class ApplicationMeetingService {
       throw new ServiceNotFoundException(`Meeting not found ${uuid}`);
     }
 
-    if (meeting.endDate && meeting.startDate > meeting.endDate) {
+    if (
+      meeting.meetingEndDate &&
+      meeting.meetingStartDate > meeting.meetingEndDate
+    ) {
       throw new ServiceValidationException(
         'Start Date must be smaller than End Date',
       );
     }
 
-    existingMeeting.applicationPaused.endDate = meeting.endDate
-      ? new Date(meeting.endDate)
-      : null;
-    existingMeeting.applicationPaused.startDate = new Date(meeting.startDate);
-    existingMeeting.description = meeting.description;
+    if (meeting.meetingStartDate) {
+      existingMeeting.meetingPause.startDate = new Date(
+        meeting.meetingStartDate,
+      );
+    }
+
+    if (meeting.description) {
+      existingMeeting.description = meeting.description;
+    }
+
+    if (meeting.meetingEndDate !== undefined) {
+      existingMeeting.meetingPause.endDate = meeting.meetingEndDate
+        ? new Date(meeting.meetingEndDate)
+        : null;
+    }
+
+    if (
+      meeting.reportStartDate !== undefined ||
+      meeting.reportEndDate !== undefined
+    ) {
```

Comment:
maybe ?
if ( meeting.reportStartDate || meeting.reportEndDate)

Reply:
If they are null I need to update the value to null, if they are undefined I dont want to change it